config/development.yaml

```diff
@@ -80,8 +80,7 @@ clusterMetadata:
     active:
       enabled: true
       initialFailoverVersion: 0
-      rpcName: "cadence-frontend"
-      rpcAddress: "localhost:7933"
+      rpcAddress: "localhost:7933" # this is to let system worker connected to the fronted end service. In cluster setup, localhost will not work
```

Comment:
nit: I think this comment is bit confusing as this address will also be used by other clusters for xdc requests. Maybe say something like: this will be used to populate publicClient rpcAddress to let system worker connect to .......

Reply:
rpcAddress is not only for xdc. 
see https://github.com/uber/cadence/blob/a58b8b9670417ccbf2cd021882e750f95d6fc508/service/worker/batcher/workflow.go#L240
https://github.com/uber/cadence/blob/a58b8b9670417ccbf2cd021882e750f95d6fc508/service/worker/parentclosepolicy/client.go#L63
https://github.com/uber/cadence/blob/a58b8b9670417ccbf2cd021882e750f95d6fc508/service/worker/parentclosepolicy/workflow.go#L114

That's why a lot of OSS users already run into issues because they use localhost for this config. 

I will make it more clear. SO system worker should include XDC worker(I thought XDC processor is still in worker service looks like i am wrong, we have move history replicator to history service). 

So 
"this is to let worker service and XDC replicator connected to the frontend service"

Reply:
yeah I understand. Sorry, probably I didn't express myself clearly. I was trying to say the original comment may make people fill the address is **ONLY** for sys worker. 
New new comment looks great! :)